packages/control-property-editor/test/unit/panels/outline/OutlinePanel.test.tsx

```diff
@@ -232,21 +235,14 @@ describe('OutlinePanel', () => {
                 controlType: 'sap.ui.comp.smarttable.SmartTable',
                 editable: true,
                 visible: true
-            },
-            {
-                name: 'ExtensionPoint',
-                controlId: '04',
-                children: [],
-                controlType: 'sap.ui.extensionpoint',
-                editable: true,
-                visible: true
             }
         ];
         const initialState: State = {
             deviceType: DeviceType.Desktop,
             scale: 1,
             outline: model,
             filterQuery: filterInitOptions,
+            scenario: 'UI_ADAPTATION',
```

Comment:
same as above

Reply:
Changed in: https://github.com/SAP/open-ux-tools/pull/1422/commits/83134d16a0fff1526e4dddf61c067912f7a5bdf2